### ISSUE TITLE
Version number for active add-ons Windows 11 

### ISSUE THREAD

**Issue by Firefoxfan** [Aris-t2/CustomCSSforFx#571]
Activating "Show version-numers of  active add-ons" results in nearly unreadable overview of all the add-ons. This problem is shown only in Windows. Linux works fine.

OS: Windows 11 Pro
Firefox 110.0.1
Graphic-Card: AMD Radeon RX500
All drivers and uzpdates actual

**Comment by Aris-t2** [comment]
Never saw or heard about such issues.

Test a clean new browser profile and only add this projects latest release to your `chrome` folder..

110.1 & 112 (Win 11)
![image](https://user-images.githubusercontent.com/6013569/224553112-ed38d400-53e0-4ef5-ab25-bd1d77c251bf.png)

**Comment by Firefoxfan** [comment]
Thank you very much for yourt hints. I uninstalled version 110 and installed 110.0.1 again, but I had big problems to install my add-ons. No symbol could be shown in the symbol-line. I had to uninstall again, went back to version 108.0.2. With that version I could install my add-ons and successfully integrated custom-css-4.3.2 with all my adaptions. After a system-backup I restarted and updated version 108.0.2 to 110.0.1 without any problems. Now the overview of my add-ons looks fine.

Thanks again.
